postgresql-config.yml.k8s_sample

```diff
@@ -38,6 +38,15 @@
             # Example:
             # collection_list: '{"postgres":{"public":{"pg_table1":["pg_index1","pg_index2"],"pg_table2":[]}}}'
             COLLECTION_LIST: '["postgres"]'
+
+            # JSON array of database names that will be ignored for metrics collection.
+            # Typically useful for cases where COLLECTION_LIST is set to 'ALL' and some databases need to be ignored.
+            # Defaults to empty '[]'.
+            # Example:
+            # COLLECTION_IGNORE_DATABASE_LIST: '["azure_maintenance","azure_sys"]'
+            #
+            # COLLECTION_IGNORE_DATABASE_LIST: '[]'
```

Comment:
I think you wanted to live this uncommented, isn't it?

Reply:
I actually commented it on propose since the default value is empty, I just leave this to be easier for the user configuring it to add the db and uncomment this

Reply:
ok!